# Test for the mean - population variance unknown

In [19]:
import pandas as pd
import math

In [20]:
openrate = pd.DataFrame(
    {
        "openrate": [
            26,
            23,
            42,
            49,
            23,
            59,
            29,
            29,
            57,
            40,
        ]
    }
)

mean_0 = 40


### Calculate tscore

In [21]:
standard_error = openrate.mean() / math.sqrt(openrate.count())
t_score = (openrate.mean() - 40) / (openrate.std() / math.sqrt(openrate.count()))
t_score

openrate   -0.529502
dtype: float64

### Search in t table

Look for `t(9,5%)` and `t(9,1%)`. 

Note:
* 9 is sample size -1 
* Take two side because now requesting *exactly* 40%

In [22]:
t_9_5 = 2.262
t_9_1 = 3.250

### Calculate P Value

Check on website: https://www.socscistatistics.com/pvalues/tdistribution.aspx

* *t-score*: `0.5295`
* *DF*: `9`
* *Significance Level*: `5%`
* `two-tailed`

In [23]:
p_value = .609269
p_value

0.609269

## Solution

`0.529502 < 2.262` Accept the null hypothesis with confidence at 5%
`0.529502 < 3.250` Accept the null hypothesis with confidence at 1%

Comment: 
> At the 5% significance level we cannot say that the competitor's open rate is not 40%
> *Different wording* The test on that sample shows that at 1% significance, our competitor's open rate is likely to be 40%.

Or looking at p-value:
* The result is not significant at p < .05.
* The result is not significant at p < .01.
